### PR TITLE
Adds XP-Pen Artist 15.6 Pro

### PR DESCRIPTION
**Pull request by tabularelf**
A friend of mine showed me OpenTabletDriver, but there was no config support for my XP-Pen Artist 15.6 Pro. So I've made a config, tested it to ensure that it's working properly.

### CODE REVIEW

**Comment by SnowLire** [comment]
This configuration format is only compatible with the 0.5 release and thus, incompatible with current master branch. Please remake with the correct format and also update TABLET.md

**Comment by tabularelf** [comment]
There isn't a TABLET.md within the master. I presume you mean TABLETS.md, which was already updated.

File: OpenTabletDriver/Configurations/XP-Pen/Artist 15.6 Pro.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "Name": "XP-Pen Artist 15.6 Pro",
```
Comment (InfinityGhost) [inline OpenTabletDriver/Configurations/XP-Pen/Artist 15.6 Pro.json:2]: Do we have a product model name that we could use for this tablet instead?

Comment (tabularelf) [reply OpenTabletDriver/Configurations/XP-Pen/Artist 15.6 Pro.json:2]: Looked around for an adequate product model name, both with the diagnostics dump and looking up the product model number online (0654913041409).
Most I could find online is the mpn. Which would be AR156PRO, according to upcindex

The diagnostics dump gives me only "UGTABLET" from the manufacturer and "15.6 inch PenDisplay" from the product name.

Comment (InfinityGhost) [reply OpenTabletDriver/Configurations/XP-Pen/Artist 15.6 Pro.json:2]: Can you dump the device strings from within the UI? That might give us a couple more clues.

**Comment by calicobean** [comment]
Hello! I don't know if this is helpful at all but I did do a dump with my 15.6 Pro, I hope this could help 
[XP-PEN 15.6 Pro Dump.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/7115637/XP-PEN.15.6.Pro.Dump.txt)

I don't know if you'd like to see this too but this is what the diagnostics told me specifically for the tablet
[XP-PEN 15.6 Pro Diagnostics.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/7115656/XP-PEN.15.6.Pro.Diagnostics.txt)
I'm very new to this so please forgive me for any confusion!

File: OpenTabletDriver/Configurations/XP-Pen/Artist 15.6 Pro.json (hunk 1 above)
Comment (InfinityGhost) [reply OpenTabletDriver/Configurations/XP-Pen/Artist 15.6 Pro.json:2]: Someone else dumped strings for me, doesn't look like there's a better model name for the tablet. Leaving it as is.

**Comment by InfinityGhost** [comment]
> Hello! I don't know if this is helpful at all but I did do a dump with my 15.6 Pro, I hope this could help
> [XP-PEN 15.6 Pro Dump.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/7115637/XP-PEN.15.6.Pro.Dump.txt)
> 
> I don't know if you'd like to see this too but this is what the diagnostics told me specifically for the tablet
> [XP-PEN 15.6 Pro Diagnostics.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/7115656/XP-PEN.15.6.Pro.Diagnostics.txt)
> I'm very new to this so please forgive me for any confusion!

You did everything I needed in order to get this verified, thanks!

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**